infrastructure/modules/container-app/variables.tf

```diff
@@ -93,6 +93,12 @@ variable "exposed_port" {
   default     = null
 }
 
+variable "liveness_probe_port" {
```

Comment:
There is no use case yet for having a different port than the default. Can we remove this and use var.port for now?